src/rpc/server/browserContextDispatcher.ts

```diff
@@ -23,17 +23,18 @@ import { PageChannel, BrowserContextChannel, BrowserContextInitializer } from '.
 import { RouteDispatcher, RequestDispatcher } from './networkDispatchers';
 import { Page } from '../../page';
 
-export class BrowserContextDispatcher extends Dispatcher<BrowserContextInitializer> implements BrowserContextChannel {
+export class BrowserContextDispatcher extends Dispatcher<BrowserContext, BrowserContextInitializer> implements BrowserContextChannel {
```

Comment:
Make it Dispatcher<BrowserContextBase...> and use this._object instead of this._context?

Reply:
I like this._context more for readability - i don't really need base